src/libs/actions/Report.ts

```diff
@@ -2386,6 +2393,13 @@ function addPolicyReport(policyReport: ReportUtils.OptimisticChatReport) {
             key: ONYXKEYS.FORMS.NEW_ROOM_FORM,
             value: {isLoading: false},
         },
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
```

Comment:
This is duplicate code of the above section. But I think we do want it in the `failureData` as `isOptimisticReport` should always clear.

Reply:
Do you see that we already have this code above though?

https://github.com/Expensify/App/blob/83267e2cf7e24e977f85a5d3a3d4483776f86aca/src/libs/actions/Report.ts#L2376-L2380